cmp/form/NumberField.js

```diff
@@ -24,22 +25,37 @@ export class NumberField extends HoistField {
         /** minimum value */
         min: PT.number,
         /** maximum value */
-        max: PT.number
+        max: PT.number,
+        /** Number of decimal places to allow on field's value, defaults to 4 */
+        precision: PT.number,
```

Comment:
`maxPrecision` ?  Don't feel strongly, I do think it's implied, but that does seem marginally more correct.